styles.css

```diff
@@ -358,11 +366,75 @@ ul {
   color: #f55800;
 }
 
-hr {
+#firstlin {
   margin-top: 180px;
   width: 80%;
 }
 
+footer {
+  background: #fff;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 120px 24px;
+  gap: 58px;
+}
+
+#foot-text {
+  width: 327px;
+  height: 216px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
+
+form {
+  width: 327px;
+  height: 330px;
+}
+
+#form-ul {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+  width: 327px;
+  height: 330px;
+  list-style-type: none;
+}
+
+#touch-but {
+  width: 131px;
+  height: 48px;
+  background: #ff6b00;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: #fff;
+  border: none;
+}
+
+.input {
+  width: 327px;
+  height: 48px;
+  border: none;
+  border-bottom: 2px solid #dbd8d7;
+}
+
+textarea {
+  width: 327px;
+  height: 114px;
+  border: none;
+  background: #e5e5e5;
+}
+
 button:hover {
   background: #fff5e1;
   border: 1px solid #f55800;
```

Comment:
The button interactions such as hover, active enable etc is not working for the form button, kindly check and fix that. :pray: